test/game_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'simplecov'
+SimpleCov.start
+
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/game"
+require "pry"
+
+class GameTest < Minitest::Test
+
+  def setup
+    @base_game = Game.new({:game_id => 123,
+                :season => 456,
+                :type => "good",
+                :date_time => "12/20/20",
+                :away_team_id => 45,
+                :home_team_id => 36,
+                :away_goals => 3,
+                :home_goals => 3,
+                :venue => "Heaven",
+                :venue_link => "venue/link"})
+
+    Game.from_csv('./data/fixtures/short_games.csv')
+    @game = Game.all[0]
+  end
+
+  def test_it_exists
+    assert_instance_of Game, @base_game
+  end
```

Comment:
I used the variable name beginning with base in order to stick with the formula from game_teams. Not sure if that is very important. If it is, then I can also update the test names so they are all following same formula.